server/config.sample.json

```diff
@@ -3,7 +3,7 @@
   "RPC_PORT": 8545,
 
   "SECRET": "jpLHCCvYF0FNXkpXKfLmS7XQcOEzFhzH0f9kL1YV8YFSdIXPAArWWrRtpliILqsH",
-  "BITLY_TOKEN": <token>,
-  "FUM_TOKEN": <token>,
-  "FUM_BASEURL": <url>
+  "BITLY_TOKEN": "dummytoken",
```

Comment:
@jvah then they should be left `null` and the parser in app say that "set them in `config.json`". Now it fails with "invalid credentials" which is stil better than parse error, though happens later.